src/devices/bus/hp_hil/hlekbd.h

```diff
@@ -6,64 +6,92 @@
 #pragma once
 
 #include "hp_hil.h"
+#include "hlebase.h"
 #include "machine/keyboard.h"
 
 
-namespace bus { namespace hp_hil {
+namespace bus {
+	namespace hp_hil {
```

Comment:
When we move to C++17, we'll actually be able to write this as `namespace bus::hp_hil {` with a single opening brace.  I know it's ugly in the mean time, but it will get better.